BreweryRatings/Models/BusinessReview.cs

```diff
@@ -9,55 +9,69 @@
 namespace ReviewNamespace
 {
     using System;
-    using System.Collections.Generic;
-
     using System.Globalization;
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
 
     public partial class BusinessReview
     {
+        private Review[] reviews;
+        private long total;
+        private string[] possibleLanguages;
+
         [JsonProperty("reviews")]
-        public Review[] Reviews { get; set; }
+        public Review[] Reviews { get => reviews; set => reviews = value; }
```

Comment:
Encapsulation is one of the four fundamental concepts in OOD, so encapsulation is good.  I'm indifferent on this change, as .net often considers {get; set;} to be a shortcut for the longer form.